doc/tutorials/klondike/app/lib/step5/components/card.dart

```diff
@@ -27,6 +27,13 @@ class Card extends PositionComponent
   final Rank rank;
   final Suit suit;
   Pile? pile;
+  final bool isBaseCard;
+  // A Base Card is rendered in outline only and is NOT playable. It can be
```

Comment:
This should be above the variable, and it can be dartdocs. (Three slashes)

Reply:
Done.